test/bdd/vdr/orb/features/orb_e2e.feature

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+@all
+@orb
+Feature: Test orb vdr
+
+  @orb_did_ops
+  Scenario Outline:
+    Then Orb DID is created through "https://localhost:48327/sidetree/v1" with key type "<keyType>" with signature suite "<signatureSuite>"
+    Then Resolve created DID through "https://localhost:48327/sidetree/v1/identifiers" and validate key type "<keyType>", signature suite "<signatureSuite>"
+    Examples:
```

Comment:
It would be good to add a BBS example and a P384 example.

Reply:
done